modules/backend/widgets/Form.php

```diff
@@ -458,6 +458,23 @@ public function onRefresh()
         return $result;
     }
 
+    /**
+     * Renders all fields of a tab in the target tab-pane.
+     *
+     * @return array
+     */
+    public function onLazyLoadTab()
+    {
+        $target  = post('target');
+        $tabName = post('name');
+
+        $fields = array_get(optional($this->getTab('primary'))->fields, $tabName);
```

Comment:
Lazy loading works only with primary tabs, throws "Invalid argument supplied for foreach()" on secondary.

Reply:
@iotch submit a new issue for that please with full details

Reply:
I'd like to hear from @tobias-kuendig if this is intended

Reply:
The original PR did not take secondary tabs into consideration. I have no idea how much effort it would be to implement the feature for them as well. 

If they re-use most of the other tab code it might be nothing more than searching the secondary tabs in the line you commented on as well.